user/src/com/google/gwt/core/server/StackTraceDeobfuscator.java

```diff
@@ -305,7 +317,13 @@ public final StackTraceElement resymbolize(StackTraceElement ste, String strongN
 
           if (declaringClass == null || declaringClass.equals(ste.getClassName())) {
             declaringClass = mappingForLine.getOriginalFile();
```

Comment:
In the linked issue if JS produces method name `<anonymous>` the `declaringClass` now becomes a filename via sourcemaps. The deobfuscated stack trace then had

```java
at com/google/gwt/core/client/impl/Impl.java.METHODNAME(com/google/gwt/core/client/impl/Impl.java:383)
```
Maybe we should try stripping `.java` and replacing `/` with `.`?

Reply:
Sure